internal/handler/web/projects.go

```diff
@@ -18,32 +18,46 @@ func (h *Handler) GetProjectsTre(ctx *gin.Context) {
 }
 
 func (h *Handler) PostProjectsTre(ctx *gin.Context) {
-	req := openapi.ProjectTRERequest{}
-	if err := bindJSONOrSetError(ctx, &req); err != nil {
+	projectTreData := openapi.ProjectTRERequest{}
+	if err := bindJSONOrSetError(ctx, &projectTreData); err != nil {
 		return
 	}
 
-	// Validate that user has owner role on the study
-	studyUUID, err := uuid.Parse(req.StudyId)
+	studyUUID, err := uuid.Parse(projectTreData.StudyId)
 	if err != nil {
 		setError(ctx, types.NewErrInvalidObject(err), "Invalid study ID")
 		return
 	}
 
 	user := middleware.GetUser(ctx)
-	requiredStudyRole := rbac.StudyRole{StudyID: studyUUID, Name: rbac.StudyOwner}
-	if hasRole, err := rbac.HasRole(user, requiredStudyRole.RoleName()); err != nil {
+	// Validate that user has owner role on the study
+	studyOwnerRole := rbac.StudyRole{StudyID: studyUUID, Name: rbac.StudyOwner}
+	if isStudyOwner, err := rbac.HasRole(user, studyOwnerRole.RoleName()); err != nil {
 		setError(ctx, err, "Failed to check study access")
 		return
-	} else if !hasRole {
+	} else if !isStudyOwner {
 		ctx.Status(http.StatusForbidden)
 		return
 	}
 
-	// TODO: Implement project creation logic
-	// validate project request roles
+	// TODO: Allow additional study admins to create projects (need to add StudyAdmin RBAC role)
 
```

Comment:
This TODO comment needs more discussion. As far as I can tell, when admins are added to a study they don't get any additional RBAC. But it looks like we want to allow study admins to make a project. Any thoughts on this one?

Reply:
this is #335

**edit**: so once they get assigned the study owner role then this will just work